README.md

```diff
@@ -26,13 +26,18 @@ To create and close all milestones listed in `config.json`:
 bot create milestone title 
 bot close milestone title
 ```
-`title` - should be in the following format `DD MMM YYYY`. For example: 13 Jan 2017
+`title` - can accept the following values:
+- any date in the format of `DD MMM YYYY`
+- `Backlog`
+- `Icebox` 
```

Comment:
@Mikanebu Does this mean that title should be one of these? Why can I not choose any title I want? Think We should be able to choose title for the milestone